crypto/armor/armor.go

```diff
@@ -8,21 +8,30 @@ import (
 	"golang.org/x/crypto/openpgp/armor" //nolint: staticcheck
 )
 
-func EncodeArmor(blockType string, headers map[string]string, data []byte) string {
+// EncodeError represents an error from calling [EncodeArmor].
+type EncodeError struct{ Err error }
```

Comment:
Please stick to the current code formatting convention for struct definitions:

```suggestion
type ErrEncode struct { 
	Err error
}
```

Same comment applies to all other error struct names.